tensorflow/python/layers/normalization_test.py

```diff
@@ -501,18 +515,32 @@ def testNoScale(self):
 
   def testRegularizers(self):
     reg = lambda x: 0.1 * math_ops.reduce_sum(x)
-    bn = normalization_layers.BatchNormalization(axis=1, beta_regularizer=reg)
-    inputs = random_ops.random_uniform((5, 4, 3), seed=1)
+    bn = normalization_layers.BatchNormalization(
+      axis=1, beta_regularizer=reg, **self.kw_dtype)
+    inputs = random_ops.random_uniform((5, 4, 3), seed=1, **self.kw_dtype)
     training = array_ops.placeholder(dtype='bool')
     _ = bn.apply(inputs, training=training)
     self.assertEqual(len(bn.losses), 1)
 
-    bn = normalization_layers.BatchNormalization(axis=1, gamma_regularizer=reg)
-    inputs = random_ops.random_uniform((5, 4, 3), seed=1)
+    bn = normalization_layers.BatchNormalization(
+      axis=1, gamma_regularizer=reg, **self.kw_dtype)
+    inputs = random_ops.random_uniform((5, 4, 3), seed=1, **self.kw_dtype)
     training = array_ops.placeholder(dtype='bool')
     _ = bn.apply(inputs, training=training)
     self.assertEqual(len(bn.losses), 1)
 
 
+class BNFloat16Test(test.TestCase):
```

Comment:
Because these classes inherit from test.TestCase and not from the test above this does nothing.

That said, see my other comment about not triplicating all tests.